scipio/src/sys/rusage.rs

```diff
@@ -0,0 +1,123 @@
+#![allow(dead_code)]
```

Comment:
Is it because there are fields in `ResourceUsage` that we don't read? It's best if we attach this to the struct, unless there is unused code all over.

Reply:
Wiped